src/libexpr/primops.cc

```diff
@@ -1490,6 +1490,18 @@ static void prim_concatLists(EvalState & state, const Pos & pos, Value * * args,
     state.concatLists(v, args[0]->listSize(), args[0]->listElems(), pos);
 }
 
+/* return a subsection of a list */
+static void prim_sublist(EvalState & state, const Pos & pos, Value * * args, Value & v)
+{
+    int start = state.forceInt(*args[0], pos);
+    size_t len = state.forceInt(*args[1], pos);
+    state.forceList(*args[2], pos);
+    size_t count = std::min(args[2]->listSize() - start, len);
```

Comment:
I think this underflows if `start` > `listSize()`.

Reply:
i can confirm it malfunctions:
```
nix-repl> builtins.sublist 11 3 [ 1 2 3 4 5 6 7 8 9 10 ]
start is 11
len is 3
listSize is 10
count is 3
[ Segmentation fault (core dumped)
```
will update the PR soon